parseAndPopulate/modules.py

```diff
@@ -179,315 +160,135 @@ def _save_file(self, save_file_dir: str):
-        result = {}
-        if self._jsons.mangled_name == 'IETFDraft':
-            index = 3
-        elif self._jsons.mangled_name == 'IETFYANGRFC':
-            return {}
-        if with_revision:
-            # try to find with revision
-            yang_file = '{}@{}.yang'.format(self.name, self.revision)
-        else:
-            # try to find without revision
-            yang_file = '{}.yang'.format(self.name)
-        try:
-            result['pyang_lint'] = self._jsons.status[self._jsons.mangled_name][yang_file][1 + index]
-            result['pyang'] = self._jsons.status[self._jsons.mangled_name][yang_file][2 + index]
-            result['confdrc'] = self._jsons.status[self._jsons.mangled_name][yang_file][3 + index]
-            result['yumadump'] = self._jsons.status[self._jsons.mangled_name][yang_file][4 + index]
-            result['yanglint'] = self._jsons.status[self._jsons.mangled_name][yang_file][5 + index]
-            return result
-        except:
-            pass
-        return {}
-
-    def _parse_document_reference(self):
-        LOGGER.debug('Parsing document reference of module {}'.format(self._path))
-        # try to find in draft without revision
-        yang_name = '{}.yang'.format(self.name)
-        yang_name_rev = '{}@{}.yang'.format(self.name, self.revision)
-        try:
-            doc_name = self._jsons.status['IETFDraft'][yang_name][0].split(
-                '</a>')[0].split('\">')[1]
-            doc_source = self._jsons.status['IETFDraft'][yang_name][0].split(
-                'a href=\"')[1].split('\">')[0]
-            return [doc_name, doc_source]
-        except KeyError:
-            pass
-        # try to find in draft with revision
-        try:
-            doc_name = self._jsons.status['IETFDraft'][yang_name_rev][
-                0].split('</a>')[0].split('\">')[1]
-            doc_source = self._jsons.status['IETFDraft'][yang_name_rev][
-                0].split('a href=\"')[1].split('\">')[0]
-            return [doc_name, doc_source]
-        except KeyError:
-            pass
-        # try to find in rfc with revision
-        try:
-            doc_name = self._jsons.status['IETFYANGRFC'][
-                yang_name_rev].split('</a>')[0].split('\">')[1]
-            doc_source = self._jsons.status['IETFYANGRFC'][
-                yang_name_rev].split('a href=\"')[1].split('\">')[0]
-            return [doc_name, doc_source]
-        except KeyError:
-            pass
-        # try to find in rfc without revision
-        try:
-            doc_name = self._jsons.status['IETFYANGRFC'][yang_name].split('</a>')[
-                0].split('\">')[1]
-            doc_source = self._jsons.status['IETFYANGRFC'][yang_name].split(
-                'a href=\"')[1].split('\">')[0]
-            return [doc_name, doc_source]
-        except KeyError:
-            pass
-        return [None, None]
+    # def _resolve_compilation_status_and_result(self, generated_from: str) -> t.Tuple[str, str]:
```

Comment:
I think it might be better to remove the commented out code. If we will need it in the future, we can always find it in git's history.

Reply:
Yes absolutely, I forgot to remove it